ConanInstall.cmake

```diff
@@ -5,8 +5,8 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   # Put it in CMAKE_BINARY_DIR so we don't end up with two when building OpenStudioApplication
   if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
```

Comment:
@jmarrec shouldn't this have our usual MD5 hash check?

Reply:
Done